test/acceptance/features/steps/steps.py

```diff
@@ -190,10 +190,38 @@ def then_sbo_jq_is(context, jq_expression, sbr_name, json_value_regex):
 
 
 # STEP
-@then(u'"{app_name}" deployment must contain SBR name "{sbr_name1}" and "{sbr_name2}"')
-def then_envFrom_contains(context, app_name, sbr_name1, sbr_name2):
+@then(u'"{app_name}" deployment must contain secret ref "{secret_ref}"')
+def then_envFrom_contains(context, app_name, secret_ref):
     time.sleep(60)
     openshift = Openshift()
     result = openshift.get_deployment_envFrom_info(app_name, context.namespace.name)
-    result | should.be_equal_to("[map[secretRef:map[name:binding-request-1]] map[secretRef:map[name:binding-request-2]]]")\
-        .desc(f'{app_name} deployment should contain secretRef: {sbr_name1} and {sbr_name2}')
+    result | should.be_equal_to(secret_ref)\
+        .desc(f'{app_name} deployment should contain secretRef: {secret_ref}')
+
+
+@given(u'Backend service "{csv_name}" CSV is installed')
+def install_csv(context, csv_name):
+    openshift = Openshift()
+    yaml_loc = "./test/acceptance/resources/backservClusterServiceVersion.yaml"
```

Comment:
`csv_name` should be used as the parameter to locate the right yaml file. Right now, not matter what we pass there, we always apply the same yaml file.

why do not we pass CSV content as the part of the step, like we do in other steps?

Reply:
> `csv_name` should be used as the parameter to locate the right yaml file. Right now, not matter what we pass there, we always apply the same yaml file.

I will rename the file, and will pass it a a variable in path.

> why do not we pass CSV content as the part of the step, like we do in other steps?
It's a file of 46 lines. Passing it directly will unnecessarily increase the length of feature file.

Reply:
that could be then the argument for all yaml content we pass. Having CSV right in feature file enables the reader to immediately see what is going on. BTW, I am pretty sure that CSV file could be reduced by omitting unnecessary parts.

Reply:
OK.